components/pg-sidecar-service/pkg/pgw/client.go

```diff
@@ -594,7 +594,6 @@ func (client *Client) DeploySqitch(db, dir, user string) error {
 	sqitchArgs := command.Args(
 		"--quiet",
 		"--top-dir", dir,
-		"--engine", "pg",
```

Comment:
Why is this removed ?

Reply:
The new version of sqitch auto-detects the db type. Engine option is not supported anymore and it gives error now